### DISCUSSION TITLE
Is there actually a validate-env command?

### DISCUSSION THREAD

**Discussion by hardyoyo** [cloudreach/sceptre#1441] [Q&A] [answered]
Hi, so far there have been two issues asking for a validate-env command, #297 and #69 both have been closed, with an implication that this command has been added to Sceptre. If this is the case, where can I find this command? If it's not the case, a comment might be added to those issues indicating why they were closed... I think right now you have a situation where folks, like me, who would like to validate their env configs, might make yet another request for this command... if there's a reason it doesn't exist, or if it's hidden in some other command, that's info that might prevent the creation of more issues you have to close.

**Comment by zaro0508** [comment, marked as answer]
I believe you can validate all templates for an environment by using the existing command `sceptre validate $ENV`.   For example `sceptre validate dev` will load all templates used by the sceptre configs in the `config/dev` folder and validate them.
